pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1437,11 +1422,6 @@ def compute_decomposition(wires):
         ]
         return decomp_ops
 
-    def adjoint(self):
-        op = SISWAP(wires=self.wires)
-        op.inverse = not self.inverse
-        return op
```

Comment:
how come these are no longer defined?

Reply:
They are defined in terms of in-place inversion.

1) This mixes the concepts of adjoint and inversion together. While the same in these cases, we shouldn't continue arbitrarily moving between the two concepts and relying on their equivalence.

2) In-place inversion will start being deprecated once we have the `Pow` wrapper.

Reply:
Got it!

If we move to a inheritance system where these operators are _known_ to be unitary, then I would say it makes sense to mix them for these specific operators